### ISSUE TITLE
Translation missing for: Leave a Comment<span class="screen-reader-text"> on %s</span> 

### ISSUE THREAD

**Issue by marcinkrzeminski** [Automattic/_s#1005]
Hi, 

I couldn't find the translation inside the generated *.pot file (based on _s.pot) for this piece:

`Leave a Comment<span class="screen-reader-text"> on %s</span>` 

located in template-tags.php. Is it missing intentionally? If not I could make a fix and make a PR. Do you accept PR? 

btw great work on _s. Keep it up! 👍

**Comment by alaczek** [comment]
The "Leave a comment" string is there in the .pot file, so I'm guessing the .pot file was not updated when changes were made to the string in `template-tags.php`. 

Feel free to submit a patch and thank you for reporting!
